app/controllers/application_controller.rb

```diff
@@ -2,13 +2,14 @@
 
 class ApplicationController < ActionController::Base
   before_action :store_request_id
+  before_action :assign_sentry_contexts
```

Comment:
Is this testable?

Reply:
Just had a look and I don't think this is tested in Publish, but I wonder if it wouldn't be worth testing ... a request spec should trigger this callback and, although it's messy, you should be able to test `Raven.tags_context`. Worth doing?

Reply:
it has tests